exporters/stringexporter/string_exporter.go

```diff
@@ -0,0 +1,41 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package stringexporter
+
+import (
+	"strconv"
+
+	"github.com/dapr/components-contrib/exporters"
+	"go.opencensus.io/trace"
+)
+
+// Metadata is the zipkin config
+type stringExporterMetadata struct {
+	Buffer *string
+}
+
+// NewStringExporter returns a new string exporter instance
+func NewStringExporter() *Exporter {
+	return &Exporter{}
+}
+
+// Exporter is an OpenCensus string exporter
+type Exporter struct {

```

Comment:
Can you rename this to `String`?

Reply:
Why? it's under the stringexporter package